src/filters/FilterNumbers.jsx

```diff
@@ -54,20 +58,21 @@ export default class FilterNumbers extends React.Component {
   render() {
     return (
       <div style={styles.base}>
+        <span style={styles.description}>{this.props.description}</span>
         <div style={{
           display: 'flex',
           alignItems: 'flex-start',
           justifyContent: 'space-between'
         }}>
-        <span style={styles.description}>{this.props.description}</span>
-        {/*
+        {
           this.props.distributions ?
           <Sparkline
-            distribution={
-              this.props.distributions[this.props[this.props.fieldName].field]
-            }/> :
+            description={this.props.description}
+            distributionMax={maxBy(this.props.distributions[this.props[this.props.fieldName].field], (d) => { return d.total })}
```

Comment:
`distributionMax={maxBy(this.props.distributions[this.props[this.props.fieldName].field], d => return d.total )}`

Same with min